spec.bs

```diff
@@ -384,30 +395,43 @@ try {
 }
 </pre>
 
-# Dependencies # {#sctn-dependencies}
+# Terminology # {#sctn-terminology}
+
+: <dfn export>SPC Credential</dfn>
+:: A WebAuthn credential that can be used for behavior defined in this spec.
+    Can be used as a normal credential (e.g. for authentication flows), as well
+    as for SPC behavior. A [=Relying Party=] must explicitly opt-in to making an
+    SPC credential during credential creation.
+
```

Comment:
I'm not sure to agree fully with the opt-in characterization. 

We've heard a desire by some (not all) for an RP to be able to use a vanilla credential for SPC when in a 1p context. In that case, I would argue that they are opting in by using the credential at authentication time. I agree that for cross-origin usage we've settled on creation-time opt-in.

I also think that this specification should not say that an SPC can be used for other authentication flows. My understanding is that we are still discussing that. At the current time I would prefer to say that this spec does not intend to constrain how SPC credentials may or may not be used in other contexts.

With all that in mind, here's a proposed change:

"When an SPC Credential is to be used by a party other than the [=Relying Party=], the [=Relying Party=] must explicitly opt in by declaring the credential for SPC at creation time. This specification does not intend to limit how SPC credentials may (or may not) be used by a [=Relying Party=] for other authentication flows (e.g., login)."

In that proposal I remain silent on RP's using vanilla WebAuthentication credentials as SPC credentials in a 1p context. Once we resolve that issue more completely we may wish to speak to it explicitly.

Reply:
I see we also say further down: "This specification does not preclude their use in other use-cases (e.g., login)." So maybe we only need to say it once (whether here or below).

Reply:
Done.